### ISSUE TITLE
Transferlist captions

### ISSUE THREAD

**Issue by blenke** [Megabit/Blazorise#5625]
Is there a way to get a caption above the transfer list Start and End? Something like this:

![image](https://github.com/user-attachments/assets/4c3c5432-bc76-4183-8db0-525fdad3c6c8)

I added it now with rows/columns/margins to get it right. But it seems logical to have something like StartCaption and EndCaption.

**Comment by stsrki** [comment]
Do you mean the "Available" and "Selected" that shows in your image?

It could be done, yes. But we need to plan for it.

**Comment by David-Moreira** [comment]
I mean, we have to take a look at it. But on first look I think the proposal is fine. If these are not provided just keep the same behaviour as is.

**Comment by blenke** [comment]
Yes indeed the 'Available' and 'Selected' texts. Perhaps a start/end captiontemplate would make it more flexible.

**Comment by almostjulian** [comment]
Sorry to necro this issue, but can this be added?

**Comment by stsrki** [comment]
Going on the list.
